src/components/pages/public/forgot-password/index.js

```diff
@@ -32,34 +56,39 @@ class ForgotPassword extends Component {
             sendEmailResetPassword,
             isLoading,
             style,
+            intl,
         } = this.props;
         return (
             <form onSubmit={handleSubmit(email => sendEmailResetPassword(email))} noValidate>
                 <section className={this.getClassName()} style={style}>
                     <Link className="rainbow-auth-firebase-forgot-password_back-link" to="/home/signin">
                         <BackIcon className="rainbow-auth-firebase-forgot-password_back-icon" />
-                        Back
+                        <FormattedMessage id="forgotpassword.back.login" defaultMessage="Back" />
                     </Link>
                     <Link to="/home">
                         <img src="/assets/rainbow-logo.svg" alt="rainbow logo" className="rainbow-auth-firebase-forgot-password_image" />
                     </Link>
-                    <p className="rainbow-auth-firebase-forgot-password_header">Reset Password</p>
+                    <p className="rainbow-auth-firebase-forgot-password_header">
+                        <FormattedMessage id="forgotpassword.title" defaultMessage="Reset password" />
+                    </p>
                     <Card className="rainbow-auth-firebase-forgot-password_card">
                         <article className="rainbow-auth-firebase-forgot-password_content">
                             <p className="rainbow-auth-firebase-forgot-password_message">
-                                A security code will be sent to your email address.
+                                <FormattedMessage
+                                    id="forgotpassword.header.text"
+                                    defaultMessage="A security code will be sent to your email address." />
                             </p>
                             <Field
                                 component={Input}
                                 name="email"
-                                label="Email Address"
+                                label={intl.formatMessage(this.translations.emailLabel)}
```

Comment:
Here you can use <FormattedMessage />.
Only use this for placeholder or other places where a span can't exists.